src/components/Asset/StakingRoyaltiesTab.tsx

```
@@ -173,6 +173,61 @@ export const StakingRoyaltiesTab: React.FC<
+
+      {(asset?.royalties?.splitRoyalties ?? []).length > 0 && (
+        <>
+          <Row span={2}>
+            <SectionTitle>Split Royalties</SectionTitle>
+          </Row>
+          {asset?.royalties?.splitRoyalties?.map((split, index) => (
+            <>
+              <Row span={2} key={index}>
+                Split Royalty {index}
+              </Row>
+              <Row>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)